.travis.yml

```diff
@@ -15,7 +15,17 @@ matrix:
     - os: linux
       rust: nightly
     - os: osx
-      osx_image: xcode11
+      osx_image: xcode11.2
+      addons:
+        chrome: stable
+        firefox: latest
+      install:
+        - curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.1/install.sh | sh
+        - source ~/.nvm/nvm.sh
+        - nvm install v10.5
```

Comment:
Is this the exact version we require here? It'd be nice to use the latest LTS (v12.13.1) if possible?

Reply:
`v10.5` came here from my copy & paste of the old working `.travis.yml`. I think it's possible to use the latest LTS (not tested).

Reply:
Let's try it. :)